Gruvo/Gruvo/ClientApp/src/app/createtweet/createtweet.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable, Inject } from "@angular/core";
+import { HttpClient, HttpErrorResponse, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import { HttpResponse } from 'selenium-webdriver/http';
+
+import { map } from 'rxjs/operators';
+
+import { IUser } from "@app/profile/user.model";
+import { ITweet } from "@app/tweet/tweet.model";
+
+@Injectable()
+export class CreateTweetService {
+  constructor(
+    @Inject('CREATETWEET_POST_TWEET_URL') private createtweetPostTweetApiURL: string,
+    private http: HttpClient) { }
+
+  postTweet(message: string) {   
+    const headers = new HttpHeaders().set('Content-Type', 'application/json');  
```

Comment:
it could be class constant as it is the same and should not be initialized on every method call